src/main/scala/io/github/pauljamescleary/petstore/infrastructure/endpoint/UserEndpoints.scala

```diff
@@ -9,20 +11,53 @@ import org.http4s.dsl.Http4sDsl
 import org.http4s.{EntityDecoder, HttpService}
 
 import scala.language.higherKinds
-import io.github.pauljamescleary.petstore.domain.{UserAlreadyExistsError, UserNotFoundError}
-import io.github.pauljamescleary.petstore.domain.users.{User, UserService}
-
-class UserEndpoints[F[_]: Effect] extends Http4sDsl[F] {
+import domain._
+import domain.users._
+import domain.authentication._
+import tsec.passwordhashers.PasswordHash
+import tsec.passwordhashers.core.PasswordHasher
 
+class UserEndpoints[F[_]: Effect, A, K] extends Http4sDsl[F] {
+//  type AuthService = AuthenticatorService[F, Long, User, K]
```

Comment:
what are these comments in here for?

Reply:
Oof, sorry for the lack of polish on this.